### PR TITLE
Fix events parsing (deciding on event identifier)

### PR DESCRIPTION
**Pull request by andreibancioiu**
For user-defined (custom) contract events, the ABI event identifier is the first topic of the event, not the actual event identifier:
 - https://github.com/multiversx/mx-chain-vm-go/blob/v1.5.27/vmhost/contexts/output.go#L270
 - https://github.com/multiversx/mx-chain-vm-go/blob/v1.5.27/vmhost/contexts/output.go#L283

Fix `TransactionEventsParser` with respect to that.

Additionally, update multisig ABI (test data), add extra tests etc.

### CODE REVIEW

File: src/smartcontracts/typesystem/abiRegistry.spec.ts [hunk 1]
```diff
@@ -63,20 +63,20 @@ describe("test abi registry", () => {
     it("binary codec correctly decodes perform action result", async () => {
         let bc = new BinaryCodec();
         let buff = Buffer.from(
-            "0588c738a5d26c0e3a2b4f9e8110b540ee9c0b71a3be057569a5a7b0fcb482c8f70000000806f05b59d3b200000000000b68656c6c6f20776f726c6400000000",
+            "0500000000000000000500d006f73c4221216fa679bc559005584c4f1160e569e1000000012a0000000003616464000000010000000107",
```
Comment (andreibancioiu) [inline src/smartcontracts/typesystem/abiRegistry.spec.ts:66]: Updated to reflect a recent multisig & multisig ABI.

File: src/transactionsOutcomeParsers/resources.ts [hunk 2]
```diff
@@ -83,6 +83,10 @@ export function findEventsByIdentifier(transactionOutcome: TransactionOutcome, i
     return findEventsByPredicate(transactionOutcome, (event) => event.identifier == identifier);
 }
 
+export function findEventsByFirstTopic(transactionOutcome: TransactionOutcome, topic: string): TransactionEvent[] {
```
Comment (andreibancioiu) [inline src/transactionsOutcomeParsers/resources.ts:86]: See https://github.com/multiversx/mx-sdk-specs/pull/61.

File: src/transactionsOutcomeParsers/transactionEventsParser.spec.ts [hunk 3]
```diff
@@ -92,6 +92,65 @@ describe("test transaction events parser", () => {
         ]);
     });
 
+    it("parses events (multisig, startPerformAction)", async function () {
```
Comment (andreibancioiu) [inline src/transactionsOutcomeParsers/transactionEventsParser.spec.ts:95]: https://devnet-explorer.multiversx.com/transactions/05d445cdd145ecb20374844dcc67f0b1e370b9aa28a47492402bc1a150c2bab4

File: src/transactionsOutcomeParsers/transactionEventsParser.ts [hunk 4]
```diff
@@ -17,9 +17,9 @@ export class TransactionEventsParser {
 
         // By default, we consider that the first topic is the event identifier.
         // This is true for log entries emitted by smart contracts:
+        // https://github.com/multiversx/mx-chain-vm-go/blob/v1.5.27/vmhost/contexts/output.go#L270
         // https://github.com/multiversx/mx-chain-vm-go/blob/v1.5.27/vmhost/contexts/output.go#L283
-        this.firstTopicIsIdentifier =
-            options.firstTopicIsIdentifier === undefined ? true : options.firstTopicIsIdentifier;
+        this.firstTopicIsIdentifier = options.firstTopicIsIdentifier ?? true;
```
Comment (andreibancioiu) [inline src/transactionsOutcomeParsers/transactionEventsParser.ts:22]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment

File: package.json [hunk 5]
```diff
@@ -10,10 +10,10 @@
   ],
   "scripts": {
     "test": "npm run tests-unit",
-    "tests-unit": "mocha $(find . -name '*.spec.ts' ! -name '*.net.spec.*')",
+    "tests-unit": "mocha $(find . -name '*.spec.ts' ! -name '*.local.net.spec.*' ! -name '*.devnet.spec.*' ! -name '*.testnet.spec.*')",
     "tests-localnet": "mocha $(find . -name '*.local.net.spec.ts')",
-    "tests-devnet": "mocha $(find . -name '*.dev.net.spec.ts')",
-    "tests-testnet": "mocha $(find . -name '*.test.net.spec.ts')",
+    "tests-devnet": "mocha $(find . -name '*.devnet.spec.ts')",
+    "tests-testnet": "mocha $(find . -name '*.testnet.spec.ts')",
```
Comment (andreibancioiu) [inline package.json:16]: Somehow rogue / unrelated change: fixed prefix for system tests. For `localnet` we didn't yet, in order to avoid merge conflicts here:


https://github.com/multiversx/mx-sdk-js-core/pull/389

**Review by danielailie — APPROVED** [review]

**Review by popenta — APPROVED** [review]
